opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -8330,7 +8335,7 @@ type ExternalReferenceEditMutations {
   contextClean: ExternalReference
   relationAdd(input: StixRefRelationshipAddInput!): StixRefRelationship
   relationDelete(fromId: StixRef!, relationship_type: String!): ExternalReference
-  askEnrichment(connectorId: ID!): Work
+  askEnrichment(connectorId: [ID!]!): [Work!]
```

Comment:
I wonder if this one is not a breaking change, do you plan to add a deprecation ?

Reply:
Nice catch. I will reintroduce the simple api and add askEnrichments